pkg/config/config.go

```diff
@@ -2,6 +2,30 @@ package config
 
 import "os"
 
+const (
+	CONFIG_FILE           = "config-file"
+	ENDPOINT              = "endpoint"
+	TOKEN                 = "token"
+	NO_HTTPS              = "no-https"
+	SHUTDOWN_HOOK_PATH    = "shutdown-hook-path"
+	DISCONNECT_AFTER_JOB  = "disconnect-after-job"
+	ENV_VARS              = "env-vars"
+	FILES                 = "files"
+	FAIL_ON_MISSING_FILES = "fail-on-missing-files"
+)
+
+var VALID_CONFIG_KEYS = []string{
+	CONFIG_FILE,
+	ENDPOINT,
+	TOKEN,
+	NO_HTTPS,
+	SHUTDOWN_HOOK_PATH,
+	DISCONNECT_AFTER_JOB,
+	ENV_VARS,
+	FILES,
+	FAIL_ON_MISSING_FILES,
```

Comment:
The convention is to use mixed caps for exported constants as well. 
https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps

example: `NoHttps`

Reply:
I've added a linter to help us catch these things automatically. 
https://github.com/semaphoreci/agent/pull/119.